Xamarin.Forms.Controls/GalleryPages/GalleryBuilder.cs

```diff
@@ -6,7 +6,8 @@ public static class GalleryBuilder
 	{
 		public static Button NavButton(string galleryName, Func<ContentPage> gallery, INavigation nav)
 		{
-			var button = new Button { Text = $"{galleryName}" };
+			var automationId = galleryName.Replace(" ", "").Replace("(", "").Replace(")", "");
```

Comment:
Maybe it's better to use a regular expression to remove all unnecessary characters at once
```suggestion
			var automationId = System.Text.RegularExpressions.Regex.Replace(galleryName, " |\\(|\\)", string.Empty);
```